src/Admin/OnboardingWizardSubmenu.php

```diff
@@ -57,5 +60,19 @@ public function register() {
 			'__return_empty_string',
 			99
 		);
+
+		if ( amp_is_legacy() && ReaderThemes::DEFAULT_READER_THEME !== AMP_Options_Manager::get_option( Option::READER_THEME ) ) {
+			$selected_theme = AMP_Options_Manager::get_option( Option::READER_THEME );
+
+			/* translators: placeholder is the name of the Reader theme. */
+			$error_message = sprintf( __( 'The Reader theme "%s" cannot be found. Your site has been updated to use the AMP Legacy Reader theme.', 'amp' ), $selected_theme );
+
+			add_settings_error(
+				AMP_Options_Manager::OPTION_NAME,
+				'unavailable_reader_theme',
+				wp_kses_post( Icon::warning()->to_html() . $error_message ),
+				'warning'
+			);
```

Comment:
I think this may need to be added as a classic `div` with a `notice notice-warning` class. This is because the settings error is getting added with every request, meaning that this dismiss button doesn't make it go away:

![image](https://user-images.githubusercontent.com/134745/89616300-019ea400-d83d-11ea-8a8a-def9bd59b59e.png)

Nevertheless, even though the notice says that the Legacy Reader theme has been selected, I'm not seeing that reflected here:

![image](https://user-images.githubusercontent.com/134745/89616512-5b06d300-d83d-11ea-9ed0-5412ec2d6938.png)

Should the heading text be updated in this case to say something like: “AMP Legacy (Fallback)”. 

The save button is also disabled, so perhaps the message should indicate the user should re-select a Reader theme to confirm their choice.

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/5159/commits/b2f4ed0f7472b9e4a4ca20946842c7e728d7ebc6.